docs/content/example_gallery.rst

```diff
@@ -104,14 +104,12 @@ Visualizing the GC content for mouse and fly genes
 
 It is well known that different species have different genome GC
 contents. Here, we used two bigWig files where the GC content was
-calculated for 50 bp windows along the genome of mice and flies and
-visualized the scores for gene regions. You can find the bigWig files in
-our `Galaxy's data
-library <http://deeptools.ie-freiburg.mpg.de/library>`__.
+calculated for 50 base windows along the genome of mice and flies and
```

Comment:
Time to split the repos into an `deeptools gallery` one and `deeptools test data`?